### PR TITLE
calibre: 2.64.0 -> 2.66.0

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change
1. Update to 2.64.0 to 2.66.0
2. Get rid of embedded python modules in favour of upstream
3. Privacy settings (based on patches from Debian)

I understand item 3 could be controversial, so I'm open to moving that to a separate PR.

**NOTE**: calibre segfaults on exit (for 2.64, 2.66 and 2.66 with patches) for me, but it doesn't seem to cause any bad behaviour.
###### Things done
- [X] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/calibre/default.nix [hunk 1]
```diff
@@ -5,25 +5,33 @@
...
 
   patches = [
-    # Patch from Debian that switches the version update change from
-    # enabled by default to disabled by default.
+    # Patches from Debian that:
+    # - disable plugin installation (very insecure)
+    # - disables loading of web bug for privacy
+    # - switches the version update from enabled to disabled by default
+    ./disable_plugins.patch
+    ./links_privacy.patch
```
Comment (grahamc) [inline pkgs/applications/misc/calibre/default.nix]: Are these patches available in a way we can `fetchpatch` them?

Comment (peterhoeg) [inline pkgs/applications/misc/calibre/default.nix]: I don't believe so. I lifted them out of the debian tarball.

Comment (fpletz) [inline pkgs/applications/misc/calibre/default.nix]: The calibre Debian packaging is managed with bazaar on launchpad here: https://code.launchpad.net/~calibre-packagers/calibre/debian

Here are the links to the patches (fixed revision):
  * https://bazaar.launchpad.net/~calibre-packagers/calibre/debian/download/head:/linksprivacy.patch-20160417214308-6hvive72pc0r4awc-1/links-privacy.patch
  * https://bazaar.launchpad.net/~calibre-packagers/calibre/debian/view/head:/debian/patches/disable_plugins.py

Comment (fpletz) [inline pkgs/applications/misc/calibre/default.nix]: Oh, and btw, to find out more information about a Debian package, the package tracker is very helpful: https://tracker.debian.org/pkg/calibre

**Comment by peterhoeg** [comment]
Thanks @fpletz, we now use ```fetchpatch``` as requested. I also bumped QT from 5.5 to 5.6 in the process. Seems to work here.

Segfault on exit is still present, but that was always the case it seems.

**Comment by grahamc** [comment]
:lgtm: and verified the segfault existed before.

---

Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/nixos/nixpkgs/18311#-:-KR1QApypZWl-tDIeWs0:bq53g9t)*

**Pull request merged**